ux/designs/integration_details/integration_details_page.md

```diff
@@ -0,0 +1,115 @@
+  - Modal for confirming user actions
+
+## Overview
+
+### Integration Details Page
+
+#### No Draft  
+
+![no draft section](img/integration_details_page_rework_11.16_no_draft.png)
+
+1. If there’s no draft for an integration. The draft section would not be shown on this page.
+
+2. Users would be able to “Create Draft” from the current version.
+
+#### Basic layout
+
+![basic layout](img/integration_details_page_rework_11.16_genericlayout.png)
+
+1. Moved status label to the top section right next to Integration Name. Using “check mark” icon to indicate this integration is active.
+
+2. Moving Draft out of the integration history. Users would be able to “Publish” and “Edit” the draft.
+
+3. Added row indicator (green bar) and checkmark icon to indicate active and running version. For active version, users would only be able to perform “Replace Draft” with the active version.
+
+![many steps and connections](img/integration_details_page_rework_11.16_many_stepsconnections.png)
+
+1. Showing an integration that has many connections and steps.
+
+#### Expanding a row
+
+![expandable row and scrolling](img/integration_details_page_rework_11.16_expand_scroll.png)
+
+1. When expanding a row item, show more details about that version.
+
+  - Start Date/Time
+  - Las Run
+  - Last Modified: user name and timestamp
+  - Modification Notes: works like commit message or note to self
+
+2. Users are able to scroll on this page. When scrolling, the whole page scrolls.
+
+
+### Interacting with the page
+
+#### Kebab menu
+
+![kebab menu](img/integration_details_page_rework_11.16_kebab.png)
+
+1. Breadcrumb & toolbar level kebab, users are able to:
+
+  - Stop Integration
+  - Replace Draft
+  - View Logs
+  - Delete Integration (all versions)
+
+  When selecting View Logs, users would be directed to the Logs page.  
+
+2. Row level kebab menu includes:
+
+  - Replace Draft
+  - Publish
+
+
+#### Modal for confirming actions
+
+![replace draft modal](img/integration_details_page_rework_11.16_replaceadraft.png)
+
+Once users click on “Replace Draft”, display a modal to notify users and to confirm the replace draft action.
+
+
+![publish version modal](img/integration_details_page_rework_11.16_publishaversion.png)
+
+If users click on “Publish”, display a modal to notify users what would happen and ask them to confirm their action.
+
+
+#### Status indicators
+
+![spinning up](img/integration_details_page_rework_11.16_spinning_up.png)
+
+1. In between state - showing the current publish version is still active, and a new version is spinning up.
+
+2. The version spinning up would have a spinner indicator.
+
+![being published](img/integration_details_page_rework_11.16_being_published.png)
+
+In this case, the published version is being “decommissioned” and the new version is being published.
+
+1. Show spinner for overall integration status.
+
+2. Show spinners for both the “decommissioning” version and the being published version.
+
```

Comment:
As an option, it would be nice to to see here would be an indicator what the progress is about and what the target stater (published / stopped) is.

E.g. the publish step actually means several 'sub-step':

* Building integration runtime
* Deploying integration runtime
* Running

Maybe we find some space to visiualize this (maybe as label below the progress meter).

Another idea: Coule we have two different progress meters: One showing the progress to starting up, and one for shutting down.

Reply:
Again, in order to keep things simple, we'd like to avoid additional information for this target user, at least until we hear feedback. If it's possible to add a hover tooltip, then we *might* be able to provide that kind of detail in the tooltip when the user hovers the cursor over the process indicator.